### PR TITLE
Add metro areas to Regions package

### PR DESCRIPTION
**Pull request by smcclure17**
Example outputs: 

```typescript
Region {
  regionId: '14460',
  fullName: 'Boston-Cambridge-Newton metro area, MA-NH',
  shortName: 'Boston-Cambridge-Newton metro',
  abbreviation: 'Boston-Cambridge-Newton metro',
  urlFragment: 'boston_cambridge_newton',
  parent: null,
  population: 4873019
}
```

```typescript
Region {
  regionId: '10420',
  fullName: 'Akron metro area, OH',
  shortName: 'Akron metro',
  abbreviation: 'Akron metro',
  urlFragment: 'akron',
  parent: null,
  population: 703479
}

```

### CODE REVIEW

File: packages/regions/src/datasets/us/metros.json [hunk 1]
```diff
@@ -0,0 +1,3155 @@
+[
+  {
+    "name": "Abilene",
+    "fipsCode": "10180",
+    "population": 172060,
+    "stateCode": ["48"],
```
Comment (smcclure17) [inline packages/regions/src/datasets/us/metros.json:6]: I imagine if we want to access the states/counties tied to each metro that we can add them as metrics

**Review by pnavarrc — APPROVED** [review]
Looks good! - Consider updating the README to describe where the regionId comes from.

File: packages/regions/src/datasets/us/metros.json (hunk 1 above)
Comment (pnavarrc) [reply packages/regions/src/datasets/us/metros.json:6]: Same for the counties, we might need to use the metro → county relationship in the future, but I think this is okay for now.

**Comment by smcclure17** [comment]
> Consider updating the README to describe where the regionId comes from.

good flag, thanks! added in 076bf273f6afc69cc8d93c6110edaffe1f349a63

**Pull request merged**